cmake/match.py

```diff
@@ -93,6 +93,12 @@ def main():
             print_incorrect_match(match_idx + 1, input_lines[input_idx].strip(), "");
             print_content(input_lines, match_lines, ignored_lines)
             sys.exit(1)
+        elif status == Status.INPUT_END:
+            # If we get to the end of the input, but still have pending matches,
```

Comment:
Maybe something like this could work:
```python
elif status == Status.INPUT_END and input_idx >= len(match_lines) + len(ignored_lines):
            print_incorrect_match(match_idx + 1, "", match_lines[match_idx]);
            print_content(input_lines, match_lines, ignored_lines)
            sys.exit(1)
```

Reply:
Yeah, that's good. Just ran into the ignored lines problem. I'll add that